src/proposals/20241017/SparkEthereum_20241017.sol

```diff
@@ -0,0 +1,160 @@
+            rateStrategyParams: Rates.RateStrategyParams({
+                optimalUsageRatio:             _bpsToRay(80_00),
+                baseVariableBorrowRate:        0,
+                variableRateSlope1:            _bpsToRay(2_00),
+                variableRateSlope2:            _bpsToRay(300_00),
+                stableRateSlope1:              0,
+                stableRateSlope2:              0,
+                baseStableRateOffset:          0,
+                stableRateExcessOffset:        0,
+                optimalStableToTotalDebtRatio: 0
+            }),
+            enabledToBorrow:       EngineFlags.DISABLED,
+            stableRateModeEnabled: EngineFlags.DISABLED,
+            borrowableInIsolation: EngineFlags.DISABLED,
+            withSiloedBorrowing:   EngineFlags.DISABLED,
+            flashloanable:         EngineFlags.DISABLED,
+            ltv:                   79_00,
+            liqThreshold:          80_00,
+            liqBonus:              5_00,
+            reserveFactor:         10_00,
+            supplyCap:             50_000_000,
+            borrowCap:             0,
+            debtCeiling:           0,
+            liqProtocolFee:        10_00,
+            eModeCategory:         0
+        });
+
+        return listings;
+    }
+
+    function priceFeedsUpdates() public pure override returns (IEngine.PriceFeedUpdate[] memory) {
+        IEngine.PriceFeedUpdate[] memory updates = new IEngine.PriceFeedUpdate[](1);
+
+        updates[0] = IEngine.PriceFeedUpdate({
+            asset:     Ethereum.SDAI,
+            priceFeed: SDAI_PRICE_FEED
+        });
+
+        return updates;
+    }
+
+    function collateralsUpdates() public pure override returns (IEngine.CollateralUpdate[] memory) {
+        // Reduce LT from 75% to 70%
+        // Reduce liquidation protocol fee from 10% to 0%
+        IEngine.CollateralUpdate[] memory updates = new IEngine.CollateralUpdate[](1);
+        updates[0] = IEngine.CollateralUpdate({
+            asset:          Ethereum.WBTC,
+            ltv:            EngineFlags.KEEP_CURRENT,
+            liqThreshold:   70_00,
+            liqBonus:       EngineFlags.KEEP_CURRENT,
+            debtCeiling:    EngineFlags.KEEP_CURRENT,
+            liqProtocolFee: 0,
+            eModeCategory:  EngineFlags.KEEP_CURRENT
+        });
+        return updates;
+    }
+
+    function _postExecute() internal override {
+        ICapAutomator capAutomator = ICapAutomator(Ethereum.CAP_AUTOMATOR);
+
+        // sUSDS onboarding
+        capAutomator.setSupplyCapConfig({
+            asset:            SUSDS,
+            max:              500_000_000,
+            gap:              50_000_000,
+            increaseCooldown: 12 hours
+        });
+
+        // Making an initial deposit right after the listing to prevent spToken value manipulation
+        IERC20(SUSDS).approve(address(LISTING_ENGINE.POOL()), 1e6);
+        LISTING_ENGINE.POOL().deposit(SUSDS, 1e6, address(this), 0);
```

Comment:
This has to be 1e18

Reply:
Good catch! Wasn't sure if 1e6 was enough to make attacks expensive enough!